src/js/svgo-worker/index.js

```diff
@@ -80,16 +84,21 @@ const pluginsData = {
   collapseGroups,
   convertPathData,
   convertTransform,
+  convertEllipseToCircle,
   removeEmptyAttrs,
   removeEmptyContainers,
   mergePaths,
   removeUnusedNS,
+  removeOffCanvasPaths,
```

Comment:
This seems to throw "__dirname is not defined" with the demo image, and leaves the app in an unusable state.

Reply:
I will look into it!